code/modules/plumbing/plumbers/grinder_chemical.dm

```diff
@@ -0,0 +1,59 @@
+/obj/machinery/plumbing/grinder_chemical
+	name = "chemical grinder"
+	desc = "chemical grinder."
+	icon_state = "grinder_chemical"
+	layer = ABOVE_ALL_MOB_LAYER
+	reagent_flags = TRANSPARENT | DRAINABLE
+	rcd_cost = 30
+	rcd_delay = 30
+	buffer = 400
+	var/eat_dir = SOUTH
+
+/obj/machinery/plumbing/grinder_chemical/Initialize(mapload, bolt)
+	. = ..()
+	AddComponent(/datum/component/plumbing/simple_supply, bolt)
+
+/obj/machinery/plumbing/grinder_chemical/can_be_rotated(mob/user, rotation_type)
```

Comment:
Shouldnt this do this in the base plumbing machinery, since all of the m can be fastened?

Reply:
Yup. I just ported the prs exactly as they were.

I can move that to /obj/machinery/plumbing if you'd like

Reply:
I think it would make more sense, although they should already be checked if they can be rotated if I am not mistaken, so shouldn't this be checked the same as the other machines?

Reply:
Sure, and yeah they are checked, but this just adds a message for why it didnt rotate afaik